### PR TITLE
Fix "value init"

### PR DESCRIPTION
**Pull request by de-nn-is**
value init fix

### CODE REVIEW

**Review by JohannaAdjana — APPROVED** [review]

**Comment by mau-seifert** [comment]
Please specify the feature/fix in the title of the PR. Possibly also the ticket like [TICKET-1] @den2702 
And if the commit messages are named "testxyz" (and don't follow a proper schema) please do a "squash" commit next time to keep the commit history clean. Under "test" you don't know immediately what has been done.

File: scripts/main.js [hunk 1]
```diff
@@ -46,6 +46,8 @@ JFCustomWidget.subscribe("ready", function () {
     jotformSettings.maxValue != '' ? document.getElementById('dat-slider').max = parseFloat(jotformSettings.maxValue.trim()) : document.getElementById('dat-slider').max = 100;
     jotformSettings.stepLength != '' ? document.getElementById('dat-slider').step = parseFloat(jotformSettings.stepLength.trim()) : document.getElementById('dat-slider').step = 10;
     jotformSettings.defaultValue != '' ? document.getElementById('dat-slider').value = parseFloat(jotformSettings.defaultValue.trim()) : document.getElementById('dat-slider').value = 50;
+    
+
```
Comment (mau-seifert) [inline scripts/main.js:50]: Remove unnecessary whitespace
